Cargo.toml

```diff
@@ -29,7 +31,8 @@ serde_json = "1"
 tokio = { version = "1.18", features = ["full"] }
 toml = "0.5"
 uint = { version = "0.9.3", default-features = false }
-workspaces = "0.6"
+# Feature `unstable` is used to enable `compile_project()`.
+workspaces = { version = "0.6", features = ["unstable"] }
```

Comment:
Are we using the `compile_project` feature?

Reply:
Not anymore, good catch! Removed the `unstable` feature in 89f17c2.